diff -up ladybird-0.0/Libraries/LibWeb/CMakeLists.txt.1~ ladybird-0.0/Libraries/LibWeb/CMakeLists.txt
--- ladybird-0.0/Libraries/LibWeb/CMakeLists.txt.1~	2025-11-03 12:23:21.000000000 +0100
+++ ladybird-0.0/Libraries/LibWeb/CMakeLists.txt	2025-11-03 14:50:32.912554077 +0100
@@ -1136,7 +1136,7 @@ ladybird_lib(LibWeb web EXPLICIT_SYMBOL_
 
 find_package(LibXml2 REQUIRED)
 
-target_link_libraries(LibWeb PRIVATE LibCore LibCompress LibCrypto LibJS LibHTTP LibGfx LibIPC LibRegex LibSyntax LibTextCodec LibUnicode LibMedia LibWasm LibXML LibIDL LibURL LibTLS LibRequests LibGC LibThreading skia ${ANGLE_TARGETS} SDL3::SDL3 LibXml2::LibXml2)
+target_link_libraries(LibWeb PRIVATE LibCore LibCompress LibCrypto LibJS LibHTTP LibGfx LibIPC LibRegex LibSyntax LibTextCodec LibUnicode LibMedia LibWasm LibXML LibIDL LibURL LibTLS LibRequests LibGC LibThreading skia ${ANGLE_TARGETS} SDL3::SDL3 LibXml2::LibXml2 GL)
 
 # FIXME: https://github.com/microsoft/vcpkg/issues/42324
 target_include_directories(LibWeb PRIVATE ${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/include)
